templates/jobservice/jobservice-dpl.yaml

```diff
@@ -44,6 +44,42 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+      - name: "change-pemission-of-directory"
```

Comment:
You don't need this init container because the security context is already set (`securityContext.fsGroup: 10000`), which means the `/var/log/jobs` volume is already owned by the harbor group.